src/components/App/App.js

```diff
@@ -0,0 +1,24 @@
+import './App.css'
+import LandingPage from '../LandingPage/LandingPage'
+import { Profile } from '../Profile/Profile'
+import { userData } from '../../UserDummyData'
+import { useState } from 'react'
+
+function App() {
+  const [user, setUser] = useState(userData)
+
+  return (
+    <div className="App">
+      <LandingPage />
+      <Profile 
+        name={user.name} 
+        email={user.email} 
+        userName={user.userName}
+        emergencyName={user.emergencyName}
+        emergencyNumber={user.emergencyNumber}
+      />
+    </div>
+  );
+}
+
+export default App;
```

Comment:
Based on some stuff I've seen we might want `App` to be a `.tsx` file, but we can figure that out later. I think this is great for now.